ddtrace/contrib/urllib3/patch.py

```diff
@@ -0,0 +1,154 @@
+from urllib import parse
+
+import urllib3
+import ddtrace
+from ddtrace import config
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+from ddtrace.http import store_request_headers, store_response_headers
+
+from ...ext import http
+from ...utils.wrappers import unwrap as _u
+from ...utils.formats import asbool, get_env
+from ...propagation.http import HTTPPropagator
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY
+
+
+# Initialize the default config vars
+config._add(
+    "urllib3",
+    {
+        "service_name": get_env("urllib3", "service_name", "urllib3"),
+        "distributed_tracing": asbool(get_env("urllib3", "distributed_tracing", True)),
+        "analytics_enabled": asbool(get_env("urllib3", "analytics_enabled", False)),
```

Comment:
Is that a pattern we use elsewhere? i.e. you have to enable App Analytics on a library by library basis? That seems strange...

Reply:
Yeah, this is very common across all of our integrations.

When you "enable app analytics" it doesn't enable for all integrations by default (by design).

Reply:
I wanted to keep the behavior across http integrations (requests/aiohttp) as close as possible. But if something has changed (such as going to GA, happy to change the default)